### ISSUE TITLE
Cursor dissapears after CTRL-C in mocha --watch

### ISSUE THREAD

**Issue by mcortesi** [mochajs/mocha#2645]
It seems mocha hides the cursor during the run but it forgots to enable it again when a SIGINT is sent...

In my script i had to do:
```js
  process.on('SIGINT', function() {
    // show cursor
    console.log("\x1B[?25h");
    process.exit();
  });
```

**Comment by rzane** [comment]
Yeah, this has been bothering me for a long time. Here's the equivalent bash:

```
trap 'echo "\x1B[?25h"' INT
```

**Comment by Munter** [comment]
This one has been bothering me a lot as well. I tried to spot what caused it a while ago, but didn't have the time to properly dive into it. It's also a hard issue to write a good test case for, so verifying a fix without regressions is a bit tricky

**Comment by refractalize** [comment]
any appetite for just keeping the cursor visible in the first place? Using tmux, it's not obvious which pane is focussed without a cursor - or is that just me?

**Comment by Munter** [comment]
I think we're not listening to enough signals that might cause an exit, so we don't call cleanup properly. We could try using https://github.com/jtlapp/node-cleanup

**Comment by jayjanssen** [comment]
Same

**Comment by mscharley** [comment]
This is happening to me too. Personally, I'm in the same situation as refractalize. I'd be more than happy if there was a flag or something to disable this altogether.

**Comment by mscharley** [comment]
Yet more alternatives, an NPM script:

```
"scripts": {
  "fix-cursor": "node -e 'console.log(\"\\x1B[?25h\");'"
}
```

**Comment by glebec** [comment]
Or an alias for your `~/.zshrc` or bash profile, lol. Thanks @mcortesi & @rzane.

```sh
alias showc='echo "\x1B[?25h"' # show cursor after e.g. Mocha has hidden it
```

**Comment by juergba** [comment]
closed by #3980

**Comment by jedwards1211** [comment]
@juergba I'm still seeing this issue in Mocha 8.2.1, it never seems to have gone away
